tests/functional/site/templates/layout.html.twig

```diff
@@ -6,7 +6,14 @@
     <script src="https://cdn.tailwindcss.com?plugins=typography"></script>
 
     <title>{% block title %}{% endblock %}</title>
-    
+
+    {% set social_image_url = block('social_image') %}
```

Comment:
This is fragile, if we don't have the block defined, the build fails, maybe we can have a fallback in some way?